test/github/kv_test.rb

```diff
@@ -54,6 +54,43 @@ def test_set_failure
     end
   end
 
+  def test_increment_default_value
+    result = @kv.increment("foo")
+
+    assert_equal 1, result
+  end
```

Comment:
Let's test that subsequent inserts work too.

Reply:
Done in b8b6ad9